documentation/v5/docs/customization.md

```diff
@@ -145,20 +145,20 @@ function CardExpiry(props) {
     return format(`${month}${year}`);
   };
 
-  return <NumberFormatBase format={_format} {...rest} />;
+  return <NumberFormatBase format={_format} {...props} />;
```

Comment:
Ohh, we need to spread both the props coming from parent and the return of usePatternFormat. Something like this.
```suggestion
  return <NumberFormatBase  {...props} format={_format} {...rest}/>;
```

Reply:
Done!